src/pages/docs/guides/chat/build-task-oriented.mdx

```diff
@@ -0,0 +1,419 @@
+See the [React UI Kit](/docs/chat/react-ui-kit) for more details.
+
+## Enriching tasks with Ably's realtime services
+
+Ably's comprehensive platform enables you to combine chat with other realtime features to create rich, interactive task experiences.
+
+**Pub/Sub channels** add interactive elements:
+- **Live polls:** Quick feedback during collaborative decisions.
+- **Status updates:** Real-time progress indicators for tasks.
+- **Interactive ratings:** Instant feedback collection.
+- **Live auctions:** Real-time bidding in marketplace scenarios.
+
+These combined services transform basic chat into comprehensive task management platforms, where communication, coordination, and real-time updates work together seamlessly.
+
+## Moderation: maintaining quality conversations
+
+Effective moderation ensures your task-oriented chats remain professional, safe, and productive. While task-oriented chats typically involve fewer participants than livestreams, maintaining conversation quality is crucial for successful task completion and keeping your users safe.
+
+Ably supports [moderating messages](/docs/chat/moderation) both before and after publication, making it easy to integrate with AI-powered or human moderation systems.
+
+* **After-publish moderation:** Messages appear instantly, then are removed if flagged as inappropriate.
+Best for most task-oriented scenarios, where immediacy often matters. It can be used to detect abusive behaviour or inappropriate content after the fact, and can also be used to remove users if necessary.
+* **Before-publish moderation:** Messages are held until approved. Use this for high-stakes tasks where every message must be vetted.
+
+### Key moderation considerations for task-oriented chat
+
+1. **Platform standards**
+   * What level of moderation is appropriate for your audience?
+   * How will you handle different types of content? For example:
+     * **Hate speech and harassment:** Detecting discriminatory language, threats, or targeted abuse.
+     * **Inappropriate content:** Flagging adult content, violence, or graphic material.
+     * **Toxicity:** Measuring overall message sentiment and hostility.
+
+2. **Technical integration**
+   * **Latency impact:** AI moderation adds up to 100ms to message delivery.
+   * **Integration options:** Choose from pre-built integrations or connect your existing moderation systems via webhooks, serverless functions, or queues.
+
+3. **Task-specific approaches**
+   * **Customer support:** Protect both customers and agents from abuse and harassment.
+   * **Gaming:** Prevent harassment while allowing enthusiastic expressions.
+   * **Marketplace:** Ensure rules are being followed such as detecting if outside-platform contact info is exchanged.
+
+### How Ably enhances task-oriented moderation
+
+Ably's flexible moderation system adapts to your task requirements:
+
+* **Per-room policies:** Different moderation rules for different task types or user roles.
```

Comment:
There's not currently a way to specify "User X is allowed to swear" except where the platform allows is (Hive Dashboard might be the only one)